### DISCUSSION TITLE
[Bug] 最近fork的项目，现在自己的项目actions里面找不到那个Enable的按钮了，无法upstream更新

### DISCUSSION THREAD

**Discussion by lukefan** [lobehub/lobehub#2352] [Self Hosting]
### 💻 Operating System

macOS

### 📦 Environment

Vercel / Zeabur / Sealos

### 🌐 Browser

Chrome

### 🐛 Bug Description

最近fork的项目，现在自己的项目actions里面找不到那个Enable的按钮了，无法upstream更新
文档里面让在自己分支的项目里面，actions页面点击enable的绿色大按钮。
现在没有了。不了解如何打开自动更新。

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment]
<img width="1693" alt="image" src="https://github.com/lobehub/lobe-chat/assets/28616219/b46bbf2f-4a8b-4ff7-87ea-fa9a6df278a5">


<img width="1728" alt="image" src="https://github.com/lobehub/lobe-chat/assets/28616219/01f185c1-63f8-47ae-a7c9-923b53c6a26e">

**Comment by lukefan** [comment]
@arvinxx 我现在看到的，是这样一个界面。
抱歉，对Github环境不熟悉。
![photo_2024-05-03_12-32-56](https://github.com/lobehub/lobe-chat/assets/3324749/aaf1b7da-8f56-4744-8543-3bd419a4c97d)

如果选择创建一个。
就会看到：
![photo_2024-05-03_12-33-57](https://github.com/lobehub/lobe-chat/assets/3324749/ec457e72-8065-432f-afe1-721e801eed6f)

**Comment by arvinxx** [comment]
你这个不是标准的 fork ，没有是正常的。点下这边这个按钮来 fork ：

<img width="1392" alt="image" src="https://github.com/lobehub/lobe-chat/assets/28616219/d9c47ae9-7e63-498c-8cf5-549d8a0f9c4a">
